src/skate.js

```diff
@@ -84,7 +84,7 @@ function skate (id, definition) {
   definition = inherit(definition || {}, skate.defaults);
   definition.id = id;
   if (definition.prototype === skate.defaults.prototype) {
-    definition.prototype = {};
+    definition.prototype = Object.create(HTMLElement.prototype);
```

Comment:
Or, perhaps more straightforward than monkeying around with prototype chains: if `definition.prototype` is provided and it is not an instance of `HTMLElement`, we copy over its own properties onto the result of `Object.create(HTMLElement.prototype)`.